courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -373,48 +299,33 @@ Examples
 Subprogram Parameter Terminology
 ----------------------------------
 
-* *Actual* parameters are values passed to a call
+* **Actual** parameters are values passed to a call
```

Comment:
Not sure about this. I typically assume bold to be emphasis, and italics to be definition/explanation. We're not emphasizing here, we're just describing what Actual and Formal mean